src/reuse/helper/errorHandler.ts

```diff
@@ -66,4 +68,37 @@ export default class ErrorHandler implements IErrorHandler {
       return "";
     }
   }
+
```

Comment:
do you want to keep the former `_retrieveFunctionNameFromStack` function or can we remove it?

Reply:
In future if we have any requirement to get the latest caller function then it will be required so we will keep.